Packages/KeyAppKit/Sources/KeyAppNetworking/HTTPClient/HTTPClient.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+/// HttpClient to handle http network requests
+public protocol IHTTPClient {
+    /// Send request to specific endpoint
+    /// - Parameters:
+    ///   - endpoint: endpoint to send request to
+    ///   - responseModel: result type of model
+    /// - Returns: specific result of `responseModel` type
+    func sendRequest<T: Decodable>(
+        endpoint: HTTPEndpoint,
+        responseModel: T.Type
+    ) async throws -> T
+}
+
+public class HTTPClient: IHTTPClient {
```

Comment:
move IHTTPClient implementation to extension

Reply:
Done